modules/curiosity.nix

```diff
@@ -24,6 +31,29 @@
         --data-dir ${(import ../.).data} \
         --scenarios-dir ${(import ../.).scenarios} \
         --log /tmp/curiosity.log
```

Comment:
Is this done on purpose?

In a systemd service, stdout is redirected to journald. Journald is providing you with some advanced log query capabilities and will manage log compression and rotation for free.

Unless I'm missing some context, I guess we should rather direct the logs to stdout instead of a text file living in `/tmp`.